common/ECSResourcesCheck/README.md

```diff
@@ -3,16 +3,20 @@
 
 Specify parameters
     
-`     ecs cluster name,
-`     
-`     aws_access_key_id,
-`     
-`     aws_secret_access_key,
-`     
-`     aws region,
-`     
-`     base json with the list of modules
-`    
+     ecs cluster name,
+     
+     aws_access_key_id,
+     
+     aws_secret_access_key,
+     
+     aws region,
+     
+     behavior - get - to get description of services/ null - to compare with base json,  
+                    
+     modules list - coma separated values of modules names,
```

Comment:
Please specify when the modules list or the base json field can be optional or required.